rs/nns/governance/src/neuron/dissolve_state_and_age.rs

```diff
@@ -20,6 +22,22 @@ pub enum DissolveStateAndAge {
 }
 
 impl DissolveStateAndAge {
+    pub fn validate(self) -> Result<Self, String> {
```

Comment:
Due to an edge case where neurons created could not be read from stable memory, we now attempt the conversion here, to ensure that the requirements are met before saving.

Reply:
Another example of vague: "an edge case". Questions that should probably be addressed: What are the conditions that lead to bad things? What bad things result from those conditions such that neurons cannot be read from stable memory?